src/compas/scene/scene.py

```diff
@@ -302,8 +306,12 @@ def redraw(self):
         if drawn_objects:
             redraw()
 
+        postdraw()
+
         return drawn_objects
 
+    redraw = draw
```

Comment:
But to be honest, I would even vote for not keep this alias, to distinct it from the redraw pluggable

Reply:
Agree